### PR TITLE
ci(dp): Test alembic migration downgrade in database migration check

### PR DESCRIPTION
**Pull request by sklgromek**
## Summary

We want to have also test for downgrades of database migrations.

### CODE REVIEW

**Review by jkmar — APPROVED** [review]

**Comment by xbend** [comment]
@magma/approvers-ci 
Check problems are fixed, change is now ready to merge and awaiting your approval.

**Review by 119Vik — APPROVED** [review]

**Pull request merged**
